release-engineering/role-handbooks/release-image-promotion.md

```diff
@@ -1,46 +1,44 @@
 # Image Promotion
 
-We need to publishing images to the `k8s-staging-kubernetes` GCS Bucket and then promoting them to production. To do that we need to follow the steps:
+When cutting a new Kubernetes release, we need to publish images to the `k8s-staging-kubernetes` GCS Bucket and then promote them to the production.
 
-This step should be done after `Official Stage` is completed and no errors.
+The Image Promotion should be done after `Official Stage` is completed and there are no errors. To do that, take the following steps:
 
- - Forking/cloning https://github.com/kubernetes-sigs/k8s-container-image-promoter
- - Forking/cloning https://github.com/kubernetes/k8s.io. If you already have that make sure you have an up-to-date copy of the repository.
- - From the root of `k8s-container-image-promoter` repository
+- Fork and clone the [`kubernetes/release` repository](https://github.com/kubernetes/release)
+- Fork the [`kubernetes/k8s.io` repository](https://github.com/kubernetes/k8s.io)
+- From the root of the `kubernetes/release` repository, run the following command to compile and install the release tools:
 
 ```shell
-go install ./cmd/cip-mm/... # Installs the manifest merge utility, cip-mm
+make release-tools
 ```
 
-- From the root of `k8s.io` repository
+- Run the following `krel` command to promote the images.  
+  - The `-i` flag runs the command in the interactive mode, so `krel` will ask you to confirm each step
+  - The `--fork` flag takes the GitHub username for a `kubernetes/k8s.io` fork that will be used to push the changes
+  - The `--tag` flag takes the version tag of the images that will be promoted
 
 ```shell
-cip-mm --base_dir=$GOPATH/src/k8s.io/k8s.io/k8s.gcr.io --staging_repo=gcr.io/k8s-staging-kubernetes --filter_tag=v1.19.0-rc.2
+krel promote-images -i --fork=<your-github-username> --tag=v1.20.0-beta.1
```

Comment:
I think we should specify here that `--tag` can be specified multiple times because...